### ISSUE TITLE
Macros not working with latest NextJS

### ISSUE THREAD

**Issue by aladinflux** [lingui/js-lingui#1192]
**Describe the bug**

The macro stopped working with NextJS 12.0.7.

**To Reproduce**
Simply start with a bank NextJS project, follow Lingui's implementation details and add the macros. Import any macro into a component.

**Expected behavior**
Macros should work correctly.

**Current behavior**
NextJS is unable to compile the page with error:
```
./pages/index.js
TypeError: Cannot read properties of undefined (reading 'uid')
```

**Additional context**

- jsLingui version ``3.13.0``
- Babel version ``7.16.7``
- Your Babel config
```
{
  "presets": ["next/babel"],
  "plugins": ["macros"]
}
```

**Comment by semoal** [comment]
Hi, I have multiple multiple projects on 12.0.7 of Next.js and they work fine with latest lingui version.
If you can provide a reproducible repo I could take a look.

Thanks

**Comment by aladinflux** [comment]
Here's there repo: https://github.com/aladinflux/lingui-issue

**Comment by semoal** [comment]
It was a bug with Next 12.0.7 but they fixed it in 12.0.8, just released a few hours ago. Give it a try, in my case it's fixed.

**Comment by benbender** [comment]
Yes, it should be fixed. Original bug was this one: vercel/nft#257
